internal/services/compute/linux_virtual_machine_resource.go

```diff
@@ -842,6 +892,19 @@ func resourceLinuxVirtualMachineRead(d *pluginsdk.ResourceData, meta interface{}
 				assessmentMode = string(patchSettings.AssessmentMode)
 			}
 			d.Set("patch_assessment_mode", assessmentMode)
+
+			bypassPlatformSafetyChecksOnUserScheduleEnabled := false
+			rebootSetting := ""
+			if patchSettings := config.PatchSettings; patchSettings != nil && patchSettings.AutomaticByPlatformSettings != nil {
+				if v := patchSettings.AutomaticByPlatformSettings.BypassPlatformSafetyChecksOnUserSchedule; v != nil {
+					bypassPlatformSafetyChecksOnUserScheduleEnabled = pointer.From(v)
```

Comment:
`pointer.From` contains nil checking so this can be condensed down to 
```suggestion
				bypassPlatformSafetyChecksOnUserScheduleEnabled = pointer.From(v)
```